bin/container_daemon

```diff
@@ -35,6 +35,7 @@ else
     btfs config Services.GuardDomain guard-interceptor-service:50051
     btfs config Services.HubDomain query-service:50051
     btfs config Services.StatusServerDomain status-grpc-service:50051
+    btfs config --bool Experimental.DisableAutoUpdate true
```

Comment:
Do you need migration for this new config for the old btfs node don't have this?
https://github.com/TRON-US/go-btfs-config/blob/master/migrations.go

Reply:
by default, it's false if it doesn't exist. so it's fine.